config/sugar/500-controller.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       serviceAccountName: eventing-controller
       enableServiceLinks: false
       containers:
-      - name: controller
+      - name: sugar-controller
```

Comment:
I prefer to not change the container names, these are mostly copy and paste between projects and it is a huge anti-pattern to rename these all over the place.

Reply:
Agree. Renaming a container to include the name of its parent object doesn't make much sense and makes it particularly annoying to fetch logs for different controllers at the same time, among other things.

This is a known limitation of the operator which I believe is being tackled right now.

Reply:
Thank you for looking at this! When looking at the current container names, there is (deployment name) => (container name)
- eventing-controller => eventing-controller
- eventing-webhook => eventing-webhook
- imc-controller => controller
- imc-dispatcher => dispatcher
- mt-broker-controller => mt-broker-controller
- mt-broker-filter => filter
- mt-broker-ingress => ingress
- sugar-controller => controller

I understand the log/consistency argument but shouldn´t then all controllers (eventing-controller, imc-controller, mt-broker-controller, sugar-controller) all use consistent naming? (either controller or the parent object´s name)

Reply:
My bad you're right. I wasn't aware those had already been renamed.

I would be in favor of waiting for the changes in the operator though. If it's only 3 components that aren't consistent (eventing-, mt-), it's still early enough to adjust them, whereas if we start renaming things using the pattern in this PR, I'm afraid that will become the norm and we should avoid that.

Reply:
We should make this change: 

- eventing-controller => controller
- eventing-webhook => webhook
- mt-broker-controller => controller